DFReader.py

```diff
@@ -605,8 +617,8 @@ class DFReader_text(DFReader):
     def __init__(self, filename, zero_time_base=False):
         DFReader.__init__(self)
         # read the whole file into memory for simplicity
-        f = open(filename, mode='r')
-        self.lines = f.readlines()
+        with open(filename, 'r') as f:
+            self.lines = f.readlines()
         f.close()
```

Comment:
Note that this is not just the timestamp which is different; the precision is also different.

tridge can say if it's OK or not.